docs/en/05_Contributing/02_Request_for_comment.md

```diff
@@ -0,0 +1,52 @@
+# Request for comment (RFC)
+
+## Why RFCs?
+This is part of the SilverStripe core decision-making process and addresses the gap between the idea phase and the pull request submission (and merge).
+
+The rationale behind this process is to:
+ * Encourage visibility on decision making
+ * Clarity on what the proposal is, its rationale and impact
+ * Reduce unnecessary work when coding large features is done without community and core-committers buy-in
+ * Improved likelihood of an optimal solution being merged
+
+The important thing to understand about the RFCs is that these are NOT a way to request features. Rather, they are a way to bring clarity so people understand what change is being proposed.
+
+## When to write an RFC?
+
+We intend RFCs to be the primary mechanisms for proposing major new features, for collecting community input on an issue, and for documenting the design decisions that have gone into SilverStripe. The RFC author is responsible for building consensus within the community and documenting dissenting opinions.
+
+Before writing the actual summary RFC document, the idea should have already had a wide range of discussion in various community communication channels. Once discussions reach a point where you think most of the difficulties have been worked through then create the RFC using the template provided.
+
+The benefits of writing an RFC for non-trivial feature proposals are:
+ * Obtaining a preliminary approval from core-committers on an architecture before code is completed, to mitigate the risk of a non-merge after a PR is submitted
+ * Community becomes aware of incoming changes prior to the implementation
+ * RFC can be used as a basis for documentation of the feature
+	
+## How to write an RFC?
+### Template
+The following heading can act as a template to starting your RFC.
+ * **Introduction** - include a reference #, title, author
+ * **Metadata** - standardised header containing at least the Author(s), Status and Version fields.
```

Comment:
Have a think about the gist idea. I think it would be good. Just needs some thought about how we would list all the gists. Perhaps using the below table in the docs? The process could be 1. Create the gist 2. Create a PR against docs with your RFC details in the table format. 3. Update the PR as the discussion progresses. 4. Once the final status of the RFC is agreed the PR is merged?

Maybe you could come up with a better way?

The RFC process started by @mateusz was just to trial out the process so we're open to input here still.

This PR is just to start sussing out a workable process we can get going on (and again would be open to change over time).